include/wx/gtk/textentry.h

```diff
@@ -63,6 +63,7 @@ class WXDLLIMPEXP_CORE wxTextEntry : public wxTextEntryBase
     bool GTKEntryOnInsertText(const char* text);
     bool GTKIsUpperCase() const { return m_isUpperCase; }
 
+    int GTKGetMaxLength() const { return m_maxlen; }
```

Comment:
Nitpicking, but also here I don't understand your use of whitespace. Why do you group the function together with a totally unrelated comment for another function?